ProjFS.Linux/PrjFSLib.Linux.Managed/VirtualizationInstance.cs

```diff
@@ -213,10 +216,18 @@ public virtual void StopVirtualizationInstance()
 
         private static string GetProcCmdline(int pid)
         {
-            using (var stream = System.IO.File.OpenText(string.Format("/proc/{0}/cmdline", pid)))
+            try
             {
-                string[] parts = stream.ReadToEnd().Split('\0');
-                return parts.Length > 0 ? parts[0] : string.Empty;
+                using (var stream = File.OpenText(string.Format("/proc/{0}/cmdline", pid)))
```

Comment:
```suggestion
                using (var stream = File.OpenText($"/proc/{pid}/cmdline"))
```

I need to start getting in the habit of using string interpolation in C# (wasn't a thing when I was using it last). Mind committing this for me while you're here?